go.mod

```diff
@@ -105,8 +105,8 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-mongodbatlas v0.4.0
 	github.com/hashicorp/vault-plugin-secrets-openldap v0.5.1
 	github.com/hashicorp/vault-plugin-secrets-terraform v0.2.0
-	github.com/hashicorp/vault/api v1.1.1
-	github.com/hashicorp/vault/sdk v0.2.1
+	github.com/hashicorp/vault/api v1.1.2-0.20210713235431-1fc8af4c041f
+	github.com/hashicorp/vault/sdk v0.2.2-0.20210713235431-1fc8af4c041f
```

Comment:
I'm curious why there is a pseudo-version? Did we not tag the tip of the release branch?

Reply:
We tagged on main, per https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/2066874670/Tagging+API+SDK+Cutting+Release+Branches

I wonder if in the future we tag on release and then bring that back to main.

Reply:
OK I think it's fine. Mainly just wondering